src/commands/transaction/transfer.ts

```diff
@@ -107,13 +106,14 @@ export default class extends ProfileCommand {
             mosaics = options.getMosaics();
         }
 
-        const message = PlainMessage.create(
-            OptionsResolver(options,
-                'message',
-                () => undefined,
-                'Introduce the message: '));
+        let message: string;
```

Comment:
Thanks! One question: what if I don't want to append any message? Is this solution still working?

Reply:
Good point @dgarcia360 should check the input and use `PlainMessage.Empty` if no message provided. 
Also assuming the cli is currently only support plain message?

Reply:
I always typed in the command line: 1.` nem2-cli transaction transfer`. I just tested another input method today: 2.`nem2-cli transaction transfer -r address -m test -c @cat.currency::0`. In mode 1, you can press the enter key to skip setting `message`, but in mode 2, an error occurs.